lib/va_profile/models/associated_person.rb

```diff
@@ -38,6 +38,14 @@ class AssociatedPerson < Base
       attribute :state, String
       attribute :zip_code, String
       attribute :primary_phone, String
+
+      # def self.bio_path
```

Comment:
Does this need to be uncommented?

Reply:
Not yet. This will be uncommented after the upgarde